pkg/blobfuseplugin/Dockerfile

```diff
@@ -12,11 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM mcr.microsoft.com/aks/fundamental/base-ubuntu:v0.0.5
-RUN wget -O /tmp/packages-microsoft-prod.deb https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb
-RUN dpkg -i /tmp/packages-microsoft-prod.deb && apt-get update && apt-get install -y ca-certificates pkg-config libfuse-dev cmake libcurl4-gnutls-dev libgnutls28-dev uuid-dev libgcrypt20-dev blobfuse
+FROM golang:1.13.10-alpine3.10 as builder
+WORKDIR /go/src/sigs.k8s.io/blobfuse-csi-driver
+ADD . .
+ARG TARGETARCH
+ARG TARGETOS
+ARG LDFLAGS
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS:--X sigs.k8s.io/blobfuse-csi-driver/pkg/blobfuse.driverVersion=latest}" -o _output/blobfuseplugin ./pkg/blobfuseplugin
+
+FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
+COPY --from=builder /go/src/sigs.k8s.io/blobfuse-csi-driver/_output/blobfuseplugin /blobfuseplugin
+RUN clean-install ca-certificates pkg-config libfuse-dev cmake libcurl4-gnutls-dev libgnutls28-dev uuid-dev libgcrypt20-dev blobfuse
```

Comment:
the CI failure said:
```
[91mE: Unable to locate package blobfuse
[0mThe command '/bin/sh -c clean-install ca-certificates pkg-config libfuse-dev cmake libcurl4-gnutls-dev libgnutls28-dev uuid-dev libgcrypt20-dev blobfuse' returned a non-zero code: 100
make[2]: *** [Makefile:91: blobfuse-container] Error 100
```

looks like `blobfuse` is not ready in this base image by `clean-install`

Reply:
@Sakuralbj can you add `RUN apt update`  before line 25 to try again?

Reply:
Ok.I will do it now